### DISCUSSION TITLE
Adding a defensive weapon/Ajout d’une arme défensif

### DISCUSSION THREAD

**Discussion by Keva75** [Crownicles/Crownicles#1125] [Idées acceptées / déjà implémentées]
**We had the right to the aegis which is an attack-oriented shield, so I ask for a defensive weapon for those who like to take players with rams !** /
On a eu le droit à l’égide qui est un bouclier axé sur l’attaque, je demande donc une arme axé sur la défensif pour ceux qui aiment prendre les joueurs à coups de bélier !

( Go complain about the translation at Google )

**Comment by BastLast** [comment]
some weapons already boost defense 
![image](https://user-images.githubusercontent.com/38099970/171295743-4344e9d9-a2be-407d-9b75-693e1578bc2f.png)

but we may add more in the future, thx for the suggestion!

**Comment by BastLast** [comment]
@quenquentin  keep that in mind for future donator items :)

**Reply by QQtin3** [reply]
Ofc np

**Comment by Pietagorh** [comment]
@Keva75 so the update of yesterday added it, it's called the Power of love, is special and gives theses stats
![image](https://user-images.githubusercontent.com/60353007/179328326-f99a0918-e3ae-4a9e-b498-61b2f4fad261.png)
